docs/reference/transformations.md

```diff
@@ -36,10 +36,10 @@ Specifying options allows to reuse a transformation in different settings.
 #### Java/Scala
 You can use Spark Dataset API in Java/Scala to define custom transformations.
 If you have a Java project, create a class that extends CustomDfTransformer or CustomDfsTransformer and implement `transform` method.
-Then use **type = ScalaClassDfTransformer** or **type = ScalaClassDfsTransformer** and configure **className** attribute.
+Then use **type = ScalaClassSparkDfTransformer** or **type = ScalaClassDfsTransformer** and configure **className** attribute.
 
 If you work without Java project, it's still possible to define your transformation in Java/Scala and compile it at runtime.
-For a 1-to-1 transformation use **type = ScalaCodeDfTransformer** and configure **code** or **file** as a function that takes `session: SparkSession, options: Map[String,String], df: DataFrame, dataObjectName: String` as parameters and returns a `DataFrame`.
+For a 1-to-1 transformation use **type = ScalaCodeSparkDfTransformer** and configure **code** or **file** as a function that takes `session: SparkSession, options: Map[String,String], df: DataFrame, dataObjectName: String` as parameters and returns a `DataFrame`.
 For many-to-many transformations use **type = ScalaCodeDfsTransformer** and configure **code** or **file** as a function that takes `session: SparkSession, options: Map[String,String], dfs: Map[String,DataFrame]` with DataFrames per input DataObject as parameter, and returns a `Map[String,DataFrame]` with the DataFrame per output DataObject.
```

Comment:
ScalaCodeDfsTransformer must be renamed to ScalaCodeSparkDfsTransformer

Reply:
thanks for the hint, changed both.

Reply:
Renaming ScalaCodeDfsTransformer to ScalaCodeSparkDfsTransformer is still open, isn't it?

Reply:
There is only one occurrence, which is in `docs/reference/transformations.md` and this is already changed.